packages/components/table/stories/table.stories.tsx

```diff
@@ -406,30 +406,25 @@ const CustomCellWithClassnamesTemplate = (args: TableProps) => {
         return (
           <div className="flex flex-col">
             <p className="text-bold text-sm capitalize">{cellValue}</p>
-            <p className="text-bold text-sm capitalize text-white/60">{user.team}</p>
+            <p className="text-bold text-sm capitalize text-default-400">{user.team}</p>
           </div>
         );
       case "status":
         return (
-          <Chip
-            className="capitalize bg-black/20 font-semibold"
-            color={statusColorMap[user.status]}
-            size="sm"
-            variant="light"
-          >
+          <Chip className="capitalize" color={statusColorMap[user.status]} size="sm" variant="flat">
```

Comment:
why is `font-semibold` removed?

Reply:
tried to make it similar to custom cells storybook style,  should I add it back?

Reply:
please add it back